### PR TITLE
PowerShell script that automate the installation building on Windows

### PR DESCRIPTION
**Pull request by idanmiara**
win-installer-builder.ps1 - added a PowerShell script that automate the installation building on Windows
win-installer.rst - added a link to the above script from the docs

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

**Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or inapplicable.**

For all pull requests:

- [X] Confirm you have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/master/CONTRIBUTING.md) 
- [X] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in documentation)
- [X] Make sure the first PR targets the master branch, eventual backports will be managed later. This can be ignored if the PR is fixing an issue that only happens in a specific branch, but not in newer ones.

The following are required only for core and extension modules (they are welcomed, but not required, for community modules):
- [ ] There is a ticket in Jira describing the issue/improvement/feature (a notable exemptions is, changes not visible to end users)
- [ ] PR for bug fixes and small new features are presented as a single commit
- [ ] Commit message must be in the form "[GEOS-XYZW] Title of the Jira ticket" (export to XML in Jira generates the message in this exact form)
- [ ] The pull request contains changes related to a single objective. If multiple focuses cannot be avoided, each one is in its own commit and has a separate ticket describing it.
- [ ] New unit tests have been added covering the changes
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] This PR passes the [QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html) (QA checks results will be reported by travis-ci after opening this PR)
- [ ] Commits changing the UI, existing user workflows, or adding new functionality, need to include documentation updates (screenshots, text)
- [ ] Commits changing the REST API, or any configuration object, should check if the REST API docs (Swagger YAML files and classic documentation) need to be updated.

### CODE REVIEW

File: doc/en/developer/source/win-installer.rst [hunk 1]
```diff
@@ -6,6 +6,8 @@ However you can create your own installer (using a Windows machine).
 
 .. note:: This step requires a Windows machine.
 
+.. note:: A Powershell script that automates the folliwing steps is avaialble `Here <https://github.com/geoserver/geoserver/edit/master/src/release/installer/win/win-installer-builder.ps1>`_.
```
Comment (aaime) [inline doc/en/developer/source/win-installer.rst:9]:
```suggestion
.. note:: A Powershell script that automates the following steps is avaialble `Here <https://github.com/geoserver/geoserver/edit/master/src/release/installer/win/win-installer-builder.ps1>`_.
```

File: src/release/installer/win/win-installer-builder.ps1 [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+$repo = "geoserver"
+$geoserver_base_name = $repo+"-"+$geoserver_ver
+$nsis_name = "nsis-"+$nsis_ver
+
+"This powershell script will compile "+$geoserver_base_name+" With "+$nsis_name+"!"
+"The instructions implented here can be found at:"
+"https://docs.geoserver.org/latest/en/developer/win-installer.html"
+
+"Clonening repo..."
```
Comment (aaime) [inline src/release/installer/win/win-installer-builder.ps1:17]:
```suggestion
"Cloning repo..."
```

File: doc/en/developer/source/win-installer.rst [hunk 3]
```diff
@@ -6,6 +6,8 @@ However you can create your own installer (using a Windows machine).
 
 .. note:: This step requires a Windows machine.
 
+.. note:: A Powershell script that automates the following steps is avaialble `Here <https://github.com/geoserver/geoserver/edit/master/src/release/installer/win/win-installer-builder.ps1>`_.
```
Comment (aaime) [inline doc/en/developer/source/win-installer.rst:9]:
```suggestion
.. note:: A community provided Powershell script that automates the following steps is available `here <https://github.com/geoserver/geoserver/edit/master/src/release/installer/win/win-installer-builder.ps1>`_.
```

**Review by aaime — APPROVED** [review]

**Comment by aaime** [comment]
Thanks, merged.

**Comment by idanmiara** [comment]
Thanks!

**Pull request merged**